src/components/list/react/ListItem.tsx

```diff
@@ -147,6 +152,7 @@ const ListItem: React.FC<ListItemProps> = ({
             onFocusCapture={preventParentFocus}
             onClick={onItemSelected}
             onKeyDown={onKeyDown()}
+            data-focus-visible-added={isHighlighted || undefined}
```

Comment:
I am not 100% sure of this change, seems a little bit hacky to use the `data-focus-visible-added` to manage the highlight. Maybe we can create another css class that has those css styles that the selector `[data-focus-visible-added]` has and then use the `isHighlighted` prop to set that class. What do you guys think?

Reply:
Have you tried using `isActive`? I think it would work for you
The List component might override the `isActive` props on ListItem but it should be fixable

Reply:
We could rename to `isHover` using a class name `list-item--hovered`